tests/baselines/reference/ES5SymbolProperty1.types

```diff
@@ -19,8 +19,8 @@ var obj = {
 }
 
 obj[Symbol.foo];
->obj[Symbol.foo] : any
->obj : { [Symbol.foo]: number; }
+>obj[Symbol.foo] : number
+>obj : { [x: string]: number; }

```

Comment:
Sad! Maybe we should continue to auto-synthesize un-declared symbols.

Reply:
Eck, no, I don't think we should. `Symbol.foo` definitely doesn't exist, so you're actually using `undefined` as a key name, if you wanna follow the coercions... That means you actually have a field by the name `"undefined"`. A string indexer is definitely more accurate than the old behavior, of manufacturing a key with a name type that doesn't actually exist.

Reply:
Yeah, never mind, I actually misread this test. Plus, for the case I was thinking of, we should consider that as a separate issue.